src/asm/symbol.cpp

```diff
@@ -563,25 +555,17 @@ void sym_SetExportAll(bool set) {
 	exportAll = set;
 }
 
-static Symbol *createBuiltinSymbol(std::string const &symName) {
-	Symbol *sym = &createsymbol(symName);
-
-	sym->isBuiltin = true;
-	sym->src = nullptr;
-	sym->fileLine = 1; // This is 0 for CLI-defined symbols
```

Comment:
This hasn't been carried over, though?

Reply:
The 0/1 distinction was only being used for printing `"<builtin>"` vs `"<command-line>"`, and it was doing so incorrectly. The `isBuiltin` flag is appropriate for this.